docs/glossary.md

```diff
@@ -16,14 +16,17 @@
 - **Strength**: The number on the dice that determines whether a hit was made. Any number less than or equal to the number is a hit.
 - **Round**: A battle round includes both sides firing at each other. A battle can consist of multiple rounds. During and at the end of each round, casualties are removed from the game and can not participate anymore.
 - **Phase**: A battle phase is subpart of a battle round where specific rules are in play that determine what the units can do. Some phases allow the units to retreat. Other phases allow the units to fire at each other.
-- **General Phase**: This phase is the default phase for units to fire. The attacker fires first and then the defender retaliates. Any casualties that the attacker hits can retaliate. At the end of the phase, all casualties are removed from the game or transformed into other units which do not participate in the battle.
-- **First Strike Phase**: This phase is for units to fire and remove casualties before the General Phase. Historically, this was used for submarines to sink sea based units as a "sneak attack". Now, any unit can participate during this phase if they have the `isFirstStrike` attribute. Units that fire during this phase will not participate during the General Phase. At the end of the phase, all casualties are removed from the game or transformed into other units which do not participate in the battle.
+- **General Combat Phase**: This phase is the default phase for units to fire. The attacker fires first and then the defender retaliates. Any casualties that the attacker hits can retaliate. At the end of the phase, all casualties are removed from the game or transformed into other units which do not participate in the battle.
+- **First Strike Combat Phase**: This phase is for units to fire and remove casualties before the General Phase. Historically, this was used for submarines to sink sea based units as a "sneak attack". Now, any unit can participate during this phase if they have the `isFirstStrike` attribute. Units that fire during this phase will not participate during the General Phase. At the end of the phase, all casualties are removed from the game or transformed into other units which do not participate in the battle.
```

Comment:
This perhaps could just be called "First Strike Combat". 

I think the test for this is to consider the sentence "That occurs during first strike combat". Does that have well defined meaning, or does it need to be qualified to know what is being referenced, eg: "That occurs during the first strike combat phase".